wondrous-app/components/Common/SidebarEntityList/index.tsx

```diff
@@ -140,6 +135,15 @@ const useSidebarData = () => {
       },
     ],
   };
+  if (orgBoard) {
+    sidebarData.data[0].items.push({
```

Comment:
Imo we can use a inline condition , the way the Collaboration item is added at the end

so in this case it would be

`!!orgBoard && {text: 'Pods' .... }`